server.js

```diff
@@ -79,6 +80,19 @@ http.listen(port, () => console.log(`Server listening on port ${port}`));
 const agent = new Agent();
 agent.addFilter(({ action }) => store.dispatch(action));
 
+//prettier-ignore
+agent.on("holdRoom", ({ action }) => {
+  if (process.env.NO_SOUND || !action.payload.hold) return;
+
+  return new Observable(notify => {
+    const audio = player.play("hotelCalifClip.wav", () => {
+      notify.complete()
+    });
+
+    return () => audio.kill()
+  })
+}, { concurrency: "cutoff"});
+
```

Comment:
Whether we use the `cutoff` mode or prefer `serial`, Observable-wrapping buys us that choice, and our rendering function remains super-simple. The return value of the Observable creation function is a function which allows `cutoff` mode (or a normal system shutdown!) to terminate the playing sound. 

> Don't start something that you can't finish!
> —Good advice